### DISCUSSION TITLE
【求助】打包网站 https://hypothes.is/

### DISCUSSION THREAD

**Discussion by yiranzhimo** [tw93/Pake#397] [Q&A] [answered]
大家好，由于电脑空间不够，无法安装 Xcode CommandLineTools，无法打包网站。

如果网友帮助打包，不胜感激！

非常感谢！

**Comment by Tlntin** [comment]
可以试试github actions方案。

**Comment by Tlntin** [comment, marked as answer]
链接在[这里](https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89)

**Reply by yiranzhimo** [reply]
非常感谢，已经搞定了🤝
